pkg/machine/ocipull/pull.go

```diff
@@ -45,11 +59,12 @@ func Pull(ctx context.Context, imageInput types.ImageReference, localDestPath *d
 	}
 
 	path, err := policyPath()
-	if err != nil {
-		return err
+	if err == nil {
+		policy, err = signature.NewPolicyFromFile(path)
+	} else {
+		logrus.Debug("no signature policy found: using no signature policy")
```

Comment:
```suggestion
		logrus.Debug("no signature policy file found: using default allow everything signature policy")
```
I think that makes it more clear what is happening to users,